### ISSUE TITLE
[BUG] Retroscanner not adjusting for 4K resolution

### ISSUE THREAD

**Issue by cloudrambler** [tsunamayo/Starship-EVO#4708]
The small circles that track ships behind the player are not correctly aligned for 4K screen resolution. The circles appear to track a smaller rectangle; perhaps 3/4 of the screen size. It's definitely different from how it was before.

Version 22w14a
GTX1070

**Comment by ProPeach** [comment]
Does this only happen when you set the resolution as 4K? Mine also aren't at the edge of the screen, I have a 1080p monitor
![image](https://user-images.githubusercontent.com/27857711/168907703-779c3750-5f8c-424f-8262-de9b8ef674f3.png)

Might have been part of the targeting rework a month or so ago that moved them all a little closer

**Comment by cloudrambler** [comment]
I have not tried other resolutions recently.

I do know that a while ago I raised a bug that these circles were not displaying at all for 4K resolution. Tsuna fixed that one. So I suspected this was a similar 4K related problem. But from what you are seeing too, it does seem to be a general issue.

In any case, the circles are much less intuitive when they are so far inset from the screen edge. If it is a deliberate change, then I vote to move them back again.

**Comment by tsunamayo** [comment]
@cloudrambler Hi, yes if it is just about the border then it is deliberate. I dont think it is more confusing, as otherwise they collide with all the other UI. It seems that plenty of game are also putting more inward too (sometime in an ellipsis). 
Maybe an ellipsis with an arrow instead of the circle would be better...

**Comment by ProPeach** [comment]
The arrow sounds nice, especially if the red arrow for your current target is larger/more obvious that the others. That can help you pick out your target very quickly as the current red circle isn't always obvious

**Comment by tsunamayo** [comment]
@ProPeach Yeah, that is just harder to code as I need to orient the widget ^^. But will give it a shot.
